### PR TITLE
Fixed verification required display

### LINKED ISSUE #1098: Financial Aid UI: Documents required for verification
Some users will need to send documents to verify income. They will receive a message in their dashboard and a short form to record when they sent their documents. 

image

## Acceptance criteria

- [x] use API in #1053 to determine whether docs are required
- [x] form for submitting date docs were sent
- [ ] after date is submitted, update message:

image

### PR DESCRIPTION
**Pull request by aliceriot**
#### What are the relevant tickets?

closes #1098 

#### What's this PR do?

This adjusts the copy a little bit and adds in a 'skip and pay full price' link.

#### How should this be manually tested?

Somehow get a financial aid going for your user with `status = 'pending-docs'`. Then go to `/dashboard`. You should see the right copy (from the issue) and a 'skip' link. The skip link should work the same way as elsewhere.

#### Screenshots (if appropriate)

payayayay

### CODE REVIEW

**Comment by noisecapella** [comment]
It looks like the skip link doesn't change the cursor when the user mouses over it, can you add a CSS rule for this?

File: static/js/components/dashboard/FinancialAidCard.js [hunk 1]
```diff
@@ -154,18 +155,20 @@ export default class FinancialAidCard extends React.Component {
...
 
         <Grid className="financial-aid-box">
           <Cell col={12}>
-            Before you can pay, you need to verify your income by mailing or faxing one
-            of the following documents:
-            <ul>
-              <li>A notarized document verifying income</li>
-            </ul>
+            Before you can pay, you need to verify your income. Please mail or fax a
+            translated and notarized income tax or income statement document.
```
Comment (noisecapella) [inline static/js/components/dashboard/FinancialAidCard.js]: To what language? We should be specific here cc @pdpinch

Comment (roberthouse54) [inline static/js/components/dashboard/FinancialAidCard.js]: How about this: "Before you can pay, you need to verify your income. Please mail or fax an English-translated and notarized income tax or income statement document."

**Comment by aliceriot** [comment]
comments addressed!

**Comment by noisecapella** [comment]
:+1:

**Pull request merged**